samigo/samigo-impl/src/java/org/sakaiproject/samigo/impl/SamigoETSProviderImpl.java

```diff
@@ -345,86 +347,34 @@ private     RenderedTemplate    getRenderedTemplate             (String template
 
         return headers;
     }
-
-    // loadTemplate from ValidationLogicImpl.java
-    /**
-     * Load and register one or more email templates (contained in the given
-     * .xml file) with the email template service
-     *
-     * @param templateFileName - the name of the .xml file to load
-     * @param templateRegistrationString - the key (name) of the template to be saved to the service
-     */
-    @SuppressWarnings("unchecked")
-    private     void                loadTemplate                    (String templateFileName, String templateRegistrationString) {
-        LOG.info(this + " loading template " + templateFileName);
-
-        SecurityAdvisor yesMan = new SecurityAdvisor() {
-            public SecurityAdvice isAllowed( String userId, String function, String reference ) {
-                return SecurityAdvice.ALLOWED;
-            }
-        };
-
-        try {
-            // Push the yesMan SA on the stack and perform the necessary actions
-            securityService.pushAdvisor(yesMan);
-
-            // Load up the resource as an input stream
-            InputStream input = SamigoETSProviderImpl.class.getClassLoader().getResourceAsStream("" + templateFileName);
-            if ( input == null ) {
-                LOG.error( "Could not load resource from '" + templateFileName + "'. Skipping ..." );
-            } else {
-                // Parse the XML, get all the child templates
-                Document document = new SAXBuilder().build( input );
-                List<?> childTemplates = document.getRootElement().getChildren( "emailTemplate" );
-                Iterator<?> iter = childTemplates.iterator();
-
-                // Create and register a template with the service for each one found in the XML file
-                while ( iter.hasNext() ) {
-                    xmlToTemplate( (Element) iter.next(), templateRegistrationString );
-                }
-            }
-        } catch (JDOMException | IOException e) {
-            LOG.error("loadTemplate error: ", e);
-        } finally { // Pop the yesMan SA off the stack (remove elevated permissions)
-            securityService.popAdvisor(yesMan);
+	
+    @Override
+    public void notifyAutoSubmitFailures(int count)
+    {
+        boolean notifyOn = serverConfigurationService.getBoolean(SamigoConstants.SAK_PROP_AUTO_SUBMIT_ERROR_NOTIFICATION_ENABLED, false);
+        if (!notifyOn)
+        {
+            return;
         }
-    }
+        String supportAddress = StringUtils.defaultIfBlank(serverConfigurationService.getString(SamigoConstants.SAK_PROP_SUPPORT_EMAIL_ADDRESS), fromAddress);
+        String toAddress = StringUtils.defaultIfBlank(serverConfigurationService.getString(SamigoConstants.SAK_PROP_AUTO_SUBMIT_ERROR_NOTIFICATION_TO_ADDRESS), supportAddress);
```

Comment:
I don't think we need to worry about the use case of a crazy Sakai admin setting null values in their sakai.properties. The scs.getString(value, default) is a super-common pattern throughout Sakai that is well understood by many developers. If we want to change null handling behavior in SCS, let's do it centrally instead of just here.